01-master.yaml

```diff
@@ -39,10 +39,11 @@ write_files:
           --api-servers=http://127.0.0.1:8080 \
           --network-plugin-dir=/etc/kubernetes/cni/net.d \
           --register-schedulable=false \
+          --cloud-provider=external \
+          --node-ip=$private_ipv4 \
           --container-runtime=docker \
           --allow-privileged=true \
           --pod-manifest-path=/etc/kubernetes/manifests \
-          --hostname-override=$private_ipv4 \
```

Comment:
Seems like this relyed on the hostname being the private IP for apiserver -> kubelet communication. This is no longer required since cloud controller manager will assign proper addresses to nodes. We're also setting `--kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname` in `kube-apiserver`